REFERENCE.md

```diff
@@ -0,0 +1,247 @@
+# Reference
+<!-- DO NOT EDIT: This document was generated by Puppet Strings -->
+
+## Resource types
+
+* [`zfs`](#zfs): Manage zfs. Create destroy and set properties on zfs instances.  **Autorequires:** If Puppet is managing the zpool at the root of this zfs in
+* [`zpool`](#zpool): Manage zpools. Create and delete zpools. The provider WILL NOT SYNC, only report differences.  Supports vdevs with mirrors, raidz, logs and s
+
+## Resource types
```

Comment:
@jbondpdx Any idea what's going on here, that the generated documentation is ending up with two `## Resource types` lines?